service/src/main/kotlin/fi/espoo/evaka/messaging/MessageNotificationEmailService.kt

```diff
@@ -42,11 +43,13 @@ class MessageNotificationEmailService(
         return tx.createQuery(
             """
             SELECT DISTINCT
+                m.thread_id,
                 mr.id as message_recipient_id,
                 p.id as person_id,
                 p.email as person_email,
                 coalesce(lower(p.language), 'fi') as language
             FROM message_recipients mr
+            JOIN message m ON m.id = mr.message_id
             JOIN message_account ma ON ma.id = mr.recipient_id 
             JOIN person p ON p.id = ma.person_id
             WHERE mr.message_id = :messageId
```

Comment:
Now that the query needs stuff from `message` it would (maybe?) make more sense to `SELECT .. FROM MESSAGE .. WHERE m.id = :messageId`